stack.go

```diff
@@ -83,19 +78,31 @@ var ErrNoFunc = errors.New("no call stack information")
 //    %+v   equivalent to %+s:%d
 //    %#v   equivalent to %#s:%d
 func (c Call) Format(s fmt.State, verb rune) {
-	if c.fn == nil {
+	frames := runtime.CallersFrames(c.pcs[:])
+	head , _ := frames.Next()
+	frame, _ := frames.Next()
```

Comment:
I'd rather see this written as:
```go
frame, more := frames.Next()
if more {
    frame, _ = frames.Next()
}
```
I don't like ignoring the first `more`. I believe it is safe with the current implementation of `frames.Next()` in Go 1.8 and Go 1.9, but it's not documented as safe, so let's not bank on it staying that way.